src/enums.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class Stock(str, Enum):
+    # todo: replace EXAMPLE stocks with meaningful names
+    EXAMPLE1: str = 'EXAMPLE1'
```

Comment:
it is enough to assign the value like this:
EXAMPLE1 = 'EXAMPLE1'

Reply:
Agreed, but wanted to keep consistency with other class attributes where we use annotations.

Reply:
Enum values are like symbolic names, so these values are not attributes in the classic sense, these are constants. Type hint should be used only where we assign values like function calls or real class attributes. We will only use these values, and never assign.

Reply:
Actually, this is my way. So you can keep it, if you want.